doc/help/CMakeLists.txt

```diff
@@ -16,5 +16,11 @@ if (RONN_LOC)
 		generate_manpage (${NAME} SECTION 7)
 	endforeach ()
 else ()
-	message (WARNING "Sorry, I cannot find ronn, thus I cannot build the man pages. Please install https://github.com/rtomayko/ronn.")
+	message ("Sorry, I cannot find ronn, thus I will install man pages directly from the repository.")
```

Comment:
I would prefer if we do not remove the link, maybe change it to:
`Sorry, I cannot find ronn (https://github.com/rtomayko/ronn), thus the script update-manpages won't work and I will install....`